crates/ra_ide/Cargo.toml

```diff
@@ -17,6 +17,12 @@ indexmap = "1.3.2"
 itertools = "0.9.0"
 log = "0.4.8"
 rustc-hash = "1.1.0"
+rand = { version = "0.7.3", features = ["small_rng"] }
+url = "*"
+maplit = "*"
+lazy_static = "*"
```

Comment:
We use `once_cell` rather than `lazy_static`

Reply:
Fixed.